tests/blackbox/test_minio_client.py

```diff
@@ -38,6 +38,7 @@
 from libs.s3.s3_blackbox_test_lib import MinIOClient
 
 
+# pylint: disable=too-many-instance-attributes
```

Comment:
Instead of skipping/disabling codacy issue try to fix/address them as much as possible.

Reply:
will address in refactor task.